app/controllers/api/autocomplete_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  class AutocompleteController < ApplicationController
+    before_action :authenticate_user!
```

Comment:
This is totally fine for for now -- down the line we may want to introduce some kind of OAuth for the API. Might add complexity for simple components that just need to query the back end really quick, though...